### DISCUSSION TITLE
What is `blockscout_homepage_explorer_url `and `api` value when verifying with Blockscout?

### DISCUSSION THREAD

**Discussion by linghuccc** [foundry-rs/foundry#8319] [Help] [answered]
According to Foundry book https://book.getfoundry.sh/reference/forge/forge-verify-contract?highlight=Blockscout#examples

Verify a contract with Blockscout should use below command:

```
5. Verify a contract with Blockscout right after deployment (make sure you add “/api?” to the end of the Blockscout homepage explorer URL):
forge create --rpc-url <rpc_https_endpoint> --private-key $devTestnetPrivateKey src/Contract.sol:SimpleStorage --verify --verifier blockscout --verifier-url <blockscout_homepage_explorer_url>/api? 
```

But what is `blockscout_homepage_explorer_url `and `api`?

For example, I have published contract `0x1A49bC432Cfd8E1494a8A3FEb58cbeF796cB9aE1 `on Artela network, when I try to verify it, it ask me to choose from 6 contract verification methods supported by Blockscout:
https://betanet-scan.artela.network/address/0x1A49bC432Cfd8E1494a8A3FEb58cbeF796cB9aE1/contract-verification

If I want to verify using command, what should I put for `blockscout_homepage_explorer_url `and `api`?

**Comment by urosognjenovic** [comment, marked as answer]
Try replacing ```<blockscout_homepage_explorer_url>/api?``` with ```https://betanet-scan.artela.network/api?```.

**Reply by linghuccc** [reply]
You are the best, man!

Didn't think simply adding "api?" could work!

**Reply by urosognjenovic** [reply]
I'm glad to have helped!

I wasn't sure about adding "/api?" as well even though it does clearly say to do so in the Foundry docs.
